schema/__tests__/sale_artwork.test.js

```diff
@@ -65,69 +62,94 @@ describe("SaleArtwork type", () => {
-            cents: 325000,
-            amount: "€3,250",
-            display: "€3,250",
-          },
-          current_bid: {
-            cents: 325000,
-            amount: "€3,250",
-            display: "€3,250",
-          },
+
+    expect(await execute(query)).toEqual({
+      sale_artwork: {
+        high_estimate: {
+          cents: 300000,
+          amount: "3,000 EUROS!",
+          display: "€3,000",
         },
-      })
+        low_estimate: {
+          cents: 200000,
+          amount: "€2,000",
+          display: "€2,000",
+        },
+        highest_bid: {
+          cents: 325000,
+          amount: "€3,250",
+          display: "€3,250",
+        },
+        current_bid: {
+          cents: 325000,
+          amount: "€3,250",
+          display: "€3,250",
+        },
+      },
     })
   })
 
-  it("can return bid increments that are above the size of a GraphQLInt", () => {
-    gravity.returns(
-      Promise.resolve({
+  describe("bid_increments", () => {
+    it("requires an increment strategy in order to retrieve increments", async () => {
+      const query = `
+        {
+          sale_artwork(id: "54c7ed2a7261692bfa910200") {
+            bid_increments
+          }
+        }
+      `
+
+      const gravityResponse = {
+        ...saleArtwork,
         minimum_next_bid_cents: 2400000000,
-      })
-    )
-    SaleArtwork.__Rewire__("gravity", gravity)
+      }
 
-    gravity
-      .onCall(1)
-      .returns(Promise.resolve({ increment_strategy: "default" }))
-      .onCall(2)
-      .returns(
-        Promise.resolve([
-          {
-            key: "default",
-            increments: [
-              {
-                from: 0,
-                to: 3000000000,
-                amount: 1000,
-              },
-            ],
-          },
-        ])
-      )
-    const query = `
-      {
-        sale_artwork(id: "54c7ed2a7261692bfa910200") {
-          bid_increments
+      const rootValue = {
+        saleLoader: () => Promise.resolve({ missing_increment_strategy: true }),
+        incrementsLoader: () => Promise.resolve(),
+      }
+
+      expect(execute(query, gravityResponse, rootValue)).rejects.toContain("Missing increment strategy")
+    })
+
+    it("can return bid increments that are above the size of a GraphQLInt", async () => {
+      const query = `
+        {
+          sale_artwork(id: "54c7ed2a7261692bfa910200") {
+            bid_increments
+          }
         }
+      `
+
+      const gravityResponse = {
+        ...saleArtwork,
+        minimum_next_bid_cents: 2400000000,
       }
-    `
-    return runQuery(query).then(data => {
+
+      const rootValue = {
+        saleLoader: () => {
+          return Promise.resolve({
+            minimum_next_bid_cents: 2400000000,
+            increment_strategy: "default",
+          })
+        },
+        incrementsLoader: sale => {
+          return Promise.resolve([
+            {
+              key: sale.increment_strategy,
```

Comment:
@alloy - Made the code a little safer in two spots. This brings up a subtle point that with our new ease in testing via simple plug-in functions that resolve values its easy to miss concrete expectations.